build-profiling-ffi.sh

```diff
@@ -35,7 +35,7 @@ case "$target" in
         remove_rpath=1
         ;;
 
-    "x86_64-apple-darwin")
+    "x86_64-apple-darwin"|"aarch64-apple-darwin")
         expected_native_static_libs=" -framework Security -framework CoreFoundation -liconv -lSystem -lresolv -lc -lm -liconv"
```

Comment:
This is interesting... Were we missing support for arm macs?

Reply:
No one so far tried compiling the lib on a mac